services/logging-operator/3.17.3/defaults/cm.yaml

```diff
@@ -97,7 +97,7 @@ data:
       image:
         # Explicitly use the default image. This should be updated when logging-operator is upgraded.
         repository: fluent/fluent-bit
-        tag: 1.8.13
+        tag: 1.8.14
```

Comment:
should we add a comment here similar to what we do in `fluent-bit` to check that these images are in sync?